src/vos/tests/vts_wal.c

```diff
@@ -1171,9 +1178,8 @@ run_wal_tests(const char *cfg)
 			akey = "uint";
 		if (is_daos_obj_type_set(otype, DAOS_OT_AKEY_LEXICAL))
 			akey = "lex";
-		dts_create_config(test_name,
-				  "WAL# Basic I/O tests dkey=%-6s akey=%s %s",
-				  dkey, akey, cfg);
+		dts_create_config(test_name, "WAL Basic IO tests dkey=%-6s akey=%s %s", dkey, akey,
```

Comment:
You can't have '/' in suite name, cmocka doesn't handle it when generating XML or Jenkins doesn't handle he result

Reply:
That's fine. Please keep the placeholder, '#', you could replace it with any char.